core/packages/test/config/launch-config.toml

```
@@ -0,0 +1,95 @@
+[settings]
+node_spawn_timeout = 120
+provider = "native"
+timeout = 600
+
+## Rococo
+[relaychain]
+chain = "rococo-local"
+default_command = "{{output_bin_dir}}/polkadot"
+default_args = ["-lparachain=debug","--beefy","--enable-offchain-indexing=true","--pruning","archive"]
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+max_validators_per_core = 2
+needed_approvals = 1
+validation_upgrade_cooldown = 2
+validation_upgrade_delay = 2
+
+[[relaychain.nodes]]
+name = "rococo-alice"
+rpc_port = 30444
+validator = true
+ws_port = 9944
+
+[[relaychain.nodes]]
+name = "rococo-bob"
+validator = true
+
+[[relaychain.nodes]]
+name = "rococo-charlie"
+validator = true
+
+[[relaychain.nodes]]
+name = "rococo-dave"
+validator = true
+
+# Snowbridge
+[[parachains]]
+add_to_genesis = true
+chain_spec_path = "{{output_dir}}/spec.json"
+cumulus_based = true
+id = 1000
+
+[[parachains.collators]]
+command = "{{output_bin_dir}}/snowbridge"
+name = "snowbrige-alice"                              
+rpc_port = 8081
+validator = true
+ws_port = 11144
+args = ["--enable-offchain-indexing=true","--pruning=archive"]
+
+[[parachains.collators.env]]
+name = "RUST_LOG"
+value = "runtime=debug,parachain=trace,cumulus-collator=trace,aura=trace,xcm=trace"
+
+[[parachains.collators]]
+command = "{{output_bin_dir}}/snowbridge"
+name = "snowbrige-bob"
+validator = true
+args = ["--enable-offchain-indexing=true","--pruning=archive"]
```

Should we add the rpc_port and ws_port values here too?
```suggestion
[[parachains.collators]]
command = "{{output_bin_dir}}/snowbridge"
name = "snowbrige-bob"
rpc_port = 8082
validator = true
ws_port = 11155
args = ["--enable-offchain-indexing=true","--pruning=archive"]
```

---

https://github.com/Snowfork/snowbridge/pull/754#issuecomment-1384709684

---

> Should we add the rpc_port and ws_port values here too?

Yes, we can explicit declaring all, but I would assume one node for each chain should be enough, normally we only connect to one node to check runtime info and others are all the same. And if you really want to connect to other node, you can just check the output of zoombienet which you can find the port to connect with.